introduction.md

```diff
@@ -10,6 +10,11 @@ This introduction introduces the term public code, explains why this is importan
 
 Public code is both computer source code (such as software and algorithms) and public policy executed in a public context, by humans or machines. Public code is explicitly distinct from regular software because it operates under fundamentally different circumstances and expectations.
 
+### Background knowledge on public code
```

Comment:
I think this section should either be the last of the public code subsections or have a lead text that speaks to why these videos are worth watching that goes a bit deeper. This mainly because in the printed version it will be weird with something like a sort of required watching in the middle of the text.

Reply:
Moved it to the end of the document in my pull request.